src/tdastro/astro_utils/unit_utils.py

```diff
@@ -27,8 +48,8 @@ def flam_to_fnu(flux_flam, wavelengths, *, wave_unit, flam_unit, fnu_unit):
     flux_fnu : `list` or `np.array`
         The flux values in fnu units.
     """
-    flux_flam = np.array(flux_flam) * flam_unit
-    wavelengths = np.array(wavelengths) * wave_unit
+    flux_flam = np.array(flux_flam)
+    wavelengths = np.array(wavelengths)
```

Comment:
Since we touch this code, let's switch to `asarray` - it doesn't copy data by default
```suggestion
    flux_flam = np.asarray(flux_flam)
    wavelengths = np.asarray(wavelengths)
```

Reply:
Good call.  I made these changes.